src/inference/src/infer_request.cpp

```diff
@@ -291,6 +291,12 @@ std::vector<VariableState> InferRequest::query_state() {
     return variable_states;
 }
 
+void InferRequest::reset_state(){OV_INFER_REQ_CALL_STATEMENT({
```

Comment:
could you please fix indentation?
```suggestion
void InferRequest::reset_state(){
    OV_INFER_REQ_CALL_STATEMENT({
```
could you please fix indentation?

Reply:
clang-format did it :)